zebra-test/src/command.rs

```diff
@@ -221,4 +224,12 @@ impl TestOutput {
         .with_section(command)
         .with_section(stdout)
     }
+
+    pub fn exit_code(&self) -> Option<i32> {
```

Comment:
Suggestion: Let's use a generic name for "code or signal". And let's document the function.

```suggestion
    /// Returns the exit signal on unix platforms, and the exit code otherwise.
    pub fn exit_status(&self) -> Option<i32> {
```